a03_sewardc.py

```diff
@@ -0,0 +1,86 @@
+#################################################################################
```

Comment:
google doc link I forgot to put it in the header:https://docs.google.com/document/d/1j8HcXLHg5q5ARNj8Zy0DKYVKwQM4TYsNa3hEvmrKkc4/edit#